includes/class-wc-cart-totals.php

```diff
@@ -876,4 +887,18 @@ protected function round_line_tax( $value ) {
 		}
 		return $value;
 	}
+
+	/**
+	 * Apply rounding to item subtotal before summing.
+	 *
+	 * @param float $value Item subtotal value.
+	 *
+	 * @return float
+	 */
+	protected function round_item_subtotal( $value ) {
```

Comment:
Can you add a @ since tag here so we know when this was introduced?

Reply:
Added